### ISSUE TITLE
Finding a better example for giving simple instructions to a computer

### ISSUE THREAD

**Issue by tutordelphia** [WilliamPWise/Prairie-World-Programming-Comics#4]
In this comic William isn't sure if the dot moving example is working. He is thinking of it with the classic example of [teaching an alien how to make a pb&j sandwich](https://gist.github.com/andersr/11004954)  ![comic](https://raw.githubusercontent.com/WilliamPWise/Prairie-World-Programming-Comics/master/Completed/01-World-of-Programming/World-of-Programming-2.jpg)

**Comment by tutordelphia** [comment]
I don't have a strong opinion but feel that the current comic is too dense and has too much content to be readable online. Any other thoughts/ideas?

**Comment by WilliamPWise** [comment]
I added scripts for an alternative introduction [here](https://github.com/WilliamPWise/Prairie-World-Programming-Comics/tree/master/Scripts) if anyone wishes to read them. Page 2 and 3 have the sandwich metaphor continuing.(Although not necessarily PB&J - let me think about that one a bit.)

**Comment by tutordelphia** [comment]
@WilliamPWise, thanks! And thanks for opening yourself to so much input and so many critiques. I know that they aren't easy and you are going out of your way to invite them. 

On the topic of critiques :-P ...One thing that I noticed about the new script and in general, is that you tend to be a bit wordy:

In this script:
> And when we see it in action, it almost looks like it’s making magic.
could be:
> In action, it looks like it works magically.
<hr>
>  But truth be told, all this magic is based on making instructions for computers to follow…
could be:
> But, all this magic is created by giving instructions to the computer...
In another medium (novel or book) this difference wouldn't matter but here I think it could make a big difference.

**Comment by tutordelphia** [comment]
Also, while the dot example may be too involved to introduce things, it may be cool as a "writing your first program" comic where the actual code is given alongside the comic. Learning how to move a "Turtle" is a great way to get started coding because you can visually see every step and you can do cool things without much code: https://repl.it/@GiddyGoat/Python-Turtle-2-Spiral

**Comment by WilliamPWise** [comment]
No problem! If anything, I hope the critiques early on will help me make better comics.

Yeah... I do tend to be very wordy, even when I try my hardest not to be - even though I repeatedly redo my wordings for lines to find what works. so any better rewording I can gladly use. Thank you for pointing those parts out.

> Also, while the dot example may be too involved to introduce things, it may be cool as a "writing your first program" comic where the actual code is given alongside the comic. Learning how to move a "Turtle" is a great way to get started coding because you can visually see every step and you can do cool things without much code:

I think I've seen that turtles program before: One source I used as a reference was [How think like a computer scientist.](http://www.ict.ru.ac.za/Resources/cspw/thinkcspy3/thinkcspy3.pdf) I think that it was though this source - and pygame - that influenced me into using that example.

I think the comic stands pretty well on its own, so it sure could be reused. Just that seems intimidating as a first impression.
